Dockerfile

```diff
@@ -13,6 +13,10 @@ RUN wget ${ARACHNI_DISTRIBUTION} -P /sectools --output-document arachni.tar.gz &
     mv arachni-${ARACHNI_LONG_VERSION} arachni && \
     rm arachni.tar.gz
 
+# healthchecks: requires curl (if not present in container)
+# RUN apk --update --no-cache add curl
```

Comment:
(see health check pull request from other scanners)

Reply:
comments removed in today's commit